packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -991,7 +973,6 @@ export function createAPIFactory(
             env,
             extensions,
             languages,
-            plugins,
```

Comment:
Question: Doesn't this change result in all extensions which use `@theia/plugin` and the respective `plugins` namespace to break?

Reply:
Potentially. The same functionality is possible by using the `extensions` export instead. I seem to remember there is a move to deprecate the Theia API and focus on the VS Code one, but may be wrong.

Options are to include this breaking change and continue tidying towards the VS Code API only or keep this here, perhaps marking it as deprecated?

Reply:
I would keep it for now, and deprecate the whole `@theia/plugin` API in one swoop later in a separate PR. Removing just one part of it sounds like inviting trouble. There might be no one actually using it, but I like to be [better safe than sorry](https://xkcd.com/1172/).

Reply:
Have now removed this removal.